### DISCUSSION TITLE
IOG- Stop export during charge period

### DISCUSSION THREAD

**Discussion by howlintj1** [springfall2008/batpred#1679] [Q&A]
Hi Just setting up predbat for Intelligent octopus go. 
Predbat is emptying my house battery before the slot at 23:30. It's charging it up to 02:00. Then doing a mix of export and then charging until the end of slot at 05:00. 

Plan here-
<img width="237" alt="image" src="https://github.com/user-attachments/assets/edcc407b-f604-4584-b0a7-bcf80215d5e0">

I'd like to get prebat to only charge during this session. Reason being for battery life I'd prefer to only do one complete cycle daily. 

I've watched a video on this and believe the setting I need to change is combine charge slots. I've turned this off. See below. 
<img width="351" alt="image" src="https://github.com/user-attachments/assets/7084fd1f-8e1b-40b4-af37-30ac73d9b0bc">

However- it didn't change the setting. Any idea in simple terms I want predbat to empty the battery by 22:29:59 and have it fall by 05:00.

**Comment by gcoan** [comment]
You want combine charge slots on,  and also turn off switch.predbat_calculate_discharge_oncharge

The latter is accidentally missing from the documentation (so I need to add it) but is I think what you need to prevent discharging in a charge slot

you could also set low power charge mode to spread the charge over the charging period rather than cramming it in at max charge rate at the beginning

**Comment by howlintj1** [comment]
Worked perfectly when set up!

Thanks. Project is brill.
